src/index.css

```diff
@@ -6,8 +6,24 @@
   font-weight: 400;
   -print-color-adjust: exact;
 
-  --red: rgb(167, 67, 67);
-  --light-grey: rgb(94, 90, 94);
-  --light-grey-transparent: rgb(94, 90, 94, 0.3);
-  --black: rgb(20, 20, 20);
+  --color-primary-500: rgb(167, 67, 67);
+  --color-neutral-100: rgb(255, 255, 255);
+  --color-neutral-200: rgb(94, 90, 94, 0.3);
+  --color-neutral-400: rgb(94, 90, 94);
+  --color-neutral-900: rgb(20, 20, 20);
+
+  --theme-color: var(--color-primary-500);
+
+  --bubble-border-color: var(--color-neutral-400);
+  --bubble-text-color: var(--color-neutral-400);
+
+  --header-line-color: var(--color-neutral-400);
+
+  --profile-lang-level-color: var(--color-neutral-400);
+  --profile-edu-school-color: var(--color-neutral-400);
+  --profile-edu-location-color: var(--color-neutral-400);
+
+  --separator-color: var(--color-neutral-200);
+
+  --title-color: var(--color-neutral-100);
```

Comment:
Those are also from the other issue. This PR should only rename the color variables and update the references to it.

Reply:
updated